zipline-loader/build.gradle.kts

```diff
@@ -48,6 +48,7 @@ kotlin {
       dependsOn(engineMain)
       dependencies {
         implementation(libs.okHttp.core)
+        implementation(libs.okHttp.mockWebServer)
```

Comment:
This should be in jniTest

Reply:
ooooh good catch. Definitely that was my intent